src/Pages/PassengerInfo/Accordian.js

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+function Accordian({ title }) {
+  const content = useRef(null);
+  const [height, setHeight] = useState(0);
+  const toggleAccordion = () => {
+    setHeight(height === 0 ? content.current.scrollHeight + 30 : 0);
+  };
+  const checkInfo = () => {};
```

Comment:
```suggestion
function Accordian({ title }) {
  const content = useRef(null);
  const [height, setHeight] = useState(0);
  
  const toggleAccordion = () => {
    setHeight(height === 0 ? content.current.scrollHeight + 30 : 0);
  };
```
사용하지 않는 함수는 삭제해주시고 구분되는 요소들끼리는 줄바꿈으로 구분해주시면 가독성 측면에서 더 좋을 것 같습니다